portal-impl/src/com/liferay/portal/linkback/LinkbackProducerUtil.java

```diff
@@ -261,6 +251,9 @@ else if (_HTTP_HEADER_VERSION_VERBOSITY_PARTIAL) {
 	private static final Log _log = LogFactoryUtil.getLog(
 		LinkbackProducerUtil.class);
 
+	private static volatile HtmlParser _htmlParser =
```

Comment:
Just to avoid that developers call things in Util classes, so instead they should use HtmlParser service. I needed to create HtmlParserUtil to be able to use method extractText in classes on portal-impl and portal-web. 
We still have some usages of this new Utill class in JSPs and classes used in JSPs, but I aligned with @tinatian that Julius will continue this work of replace it, so in the future when we can remove these usages on portal-impl, we could kill this Util class.

Reply:
@guilhermedcamacho in this case the util should match the interface method, and here it should be calling the util. Every time the ServiceProxyFactory is used, a ServiceTracker is opened, it is not necessary here, it should just reuse the one from util.

Reply:
No problem, I'm going to fix this now, thank you!